### DISCUSSION TITLE
Cannot get prices of index

### DISCUSSION THREAD

**Discussion by manichae** [joqueka/bf4py#14] [Q&A]
First of all thank you for you work.

I tried to get ISIN DE000A0DMX99
but coud not get quotes, maybe you can point me. to solution?

Thanx

**Comment by joqueka** [comment]
Hi
due to weekend I can't do much testing. Did you notice that this instrument needs mic to be set to XFRA instead of default XETR? You can get this information from `bf4py.general.instrument_information(isin='DE000A0DMX99')`. If you still have problem please provide the code that is not working.
